<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>criterion</class>
 <widget class="QWidget" name="criterion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <widget class="QToolButton" name="addCriterion">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/MainWindow/Resources/images/add.PNG</normaloff>:/MainWindow/Resources/images/add.PNG</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteCriterion">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/MainWindow/Resources/images/delete.PNG</normaloff>:/MainWindow/Resources/images/delete.PNG</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
